src/utils/models/Expense.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../../utils/db');
+const { User } = require('./User');
+
+const Expense = sequelize.define('Expense', {
+  id: {
+    type: DataTypes.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+    allowNull: false,
+  },
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  createdAt: {
+    type: DataTypes.DATE,
+    field: 'spentAt',
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
```

Comment:
I think spentAt and createdAt are different. because we can spent one hour ago but create an expense now